### PR TITLE
feat: view contact details

### PR DESCRIPTION
**Pull request by mritense**
contributes to https://ritense.tpondemand.com/entity/32564-contactgegevens-inzien

### CODE REVIEW

File: packages/user-interface/src/pages/account/account-page.tsx [hunk 1]
```diff
@@ -1,113 +1,135 @@
...
+    data: contactData,
+    loading: contactLoading,
+    error: contactError,
+    refetch: contactRefetch,
+  } = useGetBurgerProfielQuery();
+  const EMPTY_VALUE = '-';
+
+  useEffect(() => {
+    contactRefetch();
+  }, []);
```
Comment (mjcarsjens) [inline packages/user-interface/src/pages/account/account-page.tsx:20]: Is this useEffect necessary? Not entirely caught up with how Apollo GraphQL queries work exactly, but been reading up on it for this PR and I'm not sure whether this actually is necessary or not.

File: packages/user-interface/src/pages/account/account-page.tsx [hunk 2]
```diff
@@ -1,113 +1,135 @@
...
+          <FormattedMessage id="account.contactHeader" />
+        </Heading3>
+        <DetailList
+          details={[
+            {
+              headerTranslationKey: 'emailadres',
+              value:
+                contactLoading && !contactError
+                  ? ''
+                  : contactData?.getBurgerProfiel?.emailadres || EMPTY_VALUE,
```
Comment (mjcarsjens) [inline packages/user-interface/src/pages/account/account-page.tsx:40]: Is there more error handling in the default Apollo implementation? i.e. does it automatically refresh? Not sure whether it's the most user-friendly experience when there is an error to only show `-` in all fields.

**Review by mjcarsjens — COMMENTED** [review]
Looks good! Nothing mayor, a few small questions :)

**Review by bddjong — APPROVED** [review]

File: packages/localization/src/i18n/messages/en-gb.ts [hunk 3]
```diff
@@ -44,8 +44,8 @@ export const EN_GB_MESSAGES: Messages = {
     'account.edit': 'Edit',
     'account.cancel': 'Cancel',
     'account.save': 'Save',
-    'account.detail.email': 'Email address',
-    'account.detail.phone': 'Phone number',
+    'account.detail.emailadres': 'Email address',
+    'account.detail.telefoonnummer': 'Phone number',
```
[the remark this answers is not in the document]
Comment (mritense) [reply packages/localization/src/i18n/messages/en-gb.ts:48]: I based these on the property names actually received from the API (getBurgerProfiel)

File: packages/user-interface/src/pages/account/account-page.tsx (hunk 1 above)
Comment (mritense) [reply packages/user-interface/src/pages/account/account-page.tsx:20]: This makes sure fresh data is fetched each time the page is opened, to make sure cached values are not shown. Used this method throughout app and seems to work well

File: packages/user-interface/src/pages/account/account-page.tsx (hunk 2 above)
Comment (mjcarsjens) [reply packages/user-interface/src/pages/account/account-page.tsx:40]: Alright, thanks for the explanation, fine for now :)

File: packages/user-interface/src/pages/account/account-page.tsx (hunk 1 above)
Comment (mjcarsjens) [reply packages/user-interface/src/pages/account/account-page.tsx:20]: 👍

File: packages/localization/src/i18n/messages/en-gb.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (mjcarsjens) [reply packages/localization/src/i18n/messages/en-gb.ts:48]: Fair enough, solid reasoning and I agree we should be consistent with the API property names :+1:

**Review by mjcarsjens — APPROVED** [review]

**Pull request merged**